src/contributors/03-docker-development.md

```diff
@@ -29,6 +29,9 @@ Use these commands to create a custom container based on your local branch and t
 
 This is useful if you want to modify the source code of your instance to add some extra functionalities which are not available in the main release.
 
+Before you build, edit `docker/Dockerfile` and set `RUST_RELEASE_MODE=release` or the debugging will make the instance too slow.
```

Comment:
You can pass this as a [build arg](https://docs.docker.com/build/guide/build-args/). Should be like `docker build --build-arg=RUST_RELEASE_MODE=release ...`

Reply:
Like this? Not like `--build-arg RUST_RELEASE_MODE=release` ?

Reply:
@Nutomic

Reply:
Both ways seem to work.